Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -291,6 +291,17 @@ def audit_file
 
     problem "File should end with a newline" unless text.trailing_newline?
 
+    unless Dir.glob("#{formula.path.to_s.gsub(/.rb$/, "")}@*.rb").empty?
```

Comment:
Also, you probably meant `/\.rb$/`. (Not that it's likely to make a difference.)